threestudio/models/geometry/volume_grid.py

```diff
@@ -155,3 +159,18 @@ def forward_level(
         self, field: Float[Tensor, "*N 1"], threshold: float
     ) -> Float[Tensor, "*N 1"]:
         return -(field - threshold)
+
+    def export(self, points: Float[Tensor, "*N Di"], **kwargs) -> Dict[str, Any]:
+        out: Dict[str, Any] = {}
+        if self.cfg.n_feature_dims == 0:
+            return out
+        points_unscaled = points
```

Comment:
delete this line

Reply:
In `implicit-volume`, if `n_feature_dims==0` there will be no `feature_network`, so I return an empty dict. I think we better keep this to align with `implicit-volume`?